clients/line-bot-client-base/src/test/java/com/linecorp/bot/client/base/ApiClientBuilderTest.java

```diff
@@ -77,8 +79,9 @@ protected IOException buildException(Response response, MyErrorResponse errorBod
     void socksProxy() {
         int socksPort = 9020;
 
-        SocksServer socksServer = new SocksServer();
-        socksServer.start(socksPort);
+        SocksServer socksServer = new SocksServer(socksPort)
+                .setAuthenticator(new DefaultAuthenticator())
```

Comment:
From 4.1.0, it requires authenticator. If this doesn't set, test fails by NPE.

Ref: https://github.com/bbottema/java-socks-proxy-server/pull/14/files